src/workspace-data/data-table/shared/DataTable.test.ts

```diff
@@ -199,7 +199,7 @@ describe('DataTable', () => {
     // Get the checkboxes on this page
     const newPageChecks = screen.getAllByRole('checkbox', { checked: true });
     expect(newPageChecks.length).toEqual(101);
-  }, 10000);
+  }, 20000);
```

Comment:
This looks unrelated to this PR, but I've seen this test timeout multiple times, so 👍.